lib/Base/Tensor.cpp

```diff
@@ -368,6 +372,12 @@ void glow::genericTranspose(const Tensor *src, Tensor *dest,
     transposeSelectImpl(srcH, destH, shuffle);
     return;
   }
+  case ElemKind::Int8FusedQTy: {
+    auto srcH = src->getHandle<int8_t>();
+    auto destH = dest->getHandle<int8_t>();
+    transposeSelectImpl(srcH, destH, shuffle);
```

Comment:
how would this work when scale and offset is embedded into the row?
Say original tensor has two rows and 2 columns, so we have 2 rows and 4 columns to store scale and offset.
After we do transpose we mess up things. Should we disallow transpose on those tensors?

Reply:
I considered disallowing it, but thought it might make sense for a backend to prefer the data layout to be different for some weird reason (though I don't really expect it would ever make sense). Like you said you could transpose it and then have it "column-wise" fused quantized. We don't have a current use case for it though so we could always just disallow it for now.

Reply:
I made this `llvm_unreachable()` for now.